docs/floor-plan-coordinates.md

```
@@ -0,0 +1,76 @@
+  `width_m`/`length_m` directly, and not raw pixels or metres.
+- **Origin:** the top-left corner of the canvas is `(0, 0)`. `x` increases rightward,
+  `y` increases downward.
+- **Anchor point:** the top-left corner of the table's/area's own rendered bounding box
+  (not its center).
+
+### Canvas size
+
+The canvas a percentage is relative to is *derived* from the room's `width_m`/`length_m`,
+not used verbatim:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)